scripts/utils.ts

```diff
@@ -84,7 +86,20 @@ export async function setupApis(
   // is necessary for generating the SDK (as part of the user agent header).
   // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
   apis.metadata.sdkVersion = await readJsonSync(PACKAGE_JSON).version;
-  await apis.init();
+
+  // Editing the name of the Shopper Context API Model so our name is used for generating class name.
```

Comment:
Can you add a story in the backlog to fix the name collision and revert this hack. And the gus ID in the comments